lib/spack/spack/hooks/sbang.py

```diff
@@ -28,6 +28,12 @@
 else:
     system_shebang_limit = 127
 
+#: Groupdb does not exist on Windows, prevent imports
+#: on supported systems
+is_windows = str(spack.platforms.host()) == 'windows'
+if not is_windows:
+    import grp
```

Comment:
I assume this hook would fail on Windows (since it needs `grp`), but I don't see it being skipped. Are you able to install packages?

Reply:
Addressed as per discussion yesterday.